src/wp-includes/block-patterns.php

```diff
@@ -29,10 +30,9 @@ function _register_core_block_patterns_and_categories() {
 		);
 
 		foreach ( $core_block_patterns as $core_block_pattern ) {
-			register_block_pattern(
-				'core/' . $core_block_pattern,
-				require __DIR__ . '/block-patterns/' . $core_block_pattern . '.php'
-			);
+			$pattern           = require ABSPATH . WPINC . '/block-patterns/' . $core_block_pattern . '.php';
+			$pattern['source'] = 'core'; // Added in 6.3.0.
```

Comment:
The 'Added in 6.3.0' comments are useful in the Gutenberg plugin, but not so useful here, so I think they can be removed.